README.md

```diff
@@ -35,7 +35,7 @@ Unfortunately, installing from a ZIP-file is not a lot easier
 Again on Linux and possibly Mac OSX
 
     wget --content-disposition https://github.com/Oderik/xbmc-somafm/archive/master.zip
```

Comment:
Mind updating this to the new repos URL?

And I think the README has a lot of room for improvement overall!

Reply:
Sure - I'll do that in a new PR 👍🏼